src/crypto/algorithms/megolm.js

```diff
@@ -639,6 +642,7 @@ MegolmDecryption.prototype.decryptEvent = async function(event) {
             this._requestKeysForEvent(event);
         }
         throw new base.DecryptionError(
+            "OLM_UNKNOWN_MESSAGE_INDEX",
```

Comment:
The error is not necessarily `OLM_UNKNOWN_MESSAGE_INDEX` here, or at least there is a conditional above that checks for it so I would assume not.

Reply:
IRL, me and @lampholder discussed. We don't especially care about all of the errors thrown by `decryptGroupMessage` (all of which are plain old `Error`) so I'll just rename this to `OLM_GROUP_MESSAGE_ERROR`.

Reply:
(I changed my mind, we can still emit OLM_UNKNOWN_MESSAGE_INDEX)